bldsys/cmake/sample_helper.cmake

```diff
@@ -181,4 +199,36 @@ endif()
     if(VKB_DO_CLANG_TIDY)
         set_target_properties(${PROJECT_NAME} PROPERTIES CXX_CLANG_TIDY "${VKB_DO_CLANG_TIDY}")
     endif()
+
+    if(DEFINED Vulkan_dxc_EXECUTABLE AND DEFINED SHADERS_HLSL)
+        compile_hlsl_shaders()
+    endif()
+endfunction()
+
+function(compile_hlsl_shaders)
```

Comment:
Wouldn't it be cleaner, if the shaders to be handled would be passed into this function as arguments?

Reply:
I made a change a you suggested, tell me if now it is okay.